sdk/src/Handlers/AwsSdk/Internal/XRayPipelineHandler.cs

```diff
@@ -451,6 +459,14 @@ private void ProcessException(AmazonServiceException ex, Entity subsegment)
             _recorder.AddHttpInformation("response", responseAttributes);
 
             subsegment.Aws["request_id"] = ex.RequestId;
+
+            // Try to obtain x-amz-id-2 from s3 exception
+            // https://github.com/aws/aws-sdk-net/blob/master/sdk/src/Services/S3/Custom/AmazonS3Exception.cs#L125
```

Comment:
It's usually better to provide a short description rather than a link in such cases. The code in the link might change in the future and may point to something else entirely.

Reply:
Will update